baba_jaga.md

```diff
@@ -0,0 +1,32 @@
+# Baba Jaga
+
+## Einleitung
+
+Baba Jaga ist ein altes Familienrezept für einfache Pfannkuchen, die im Wesentlichen satt machen sollen. Besonderes Merkmal dieser Pfannkuchen ist, dass sie aus mehreren Lagen bestehen. Normalerweise wird nur ein einzelner Pfannkuchen zubereitet, der die gesamte Pfanne ausfüllt. Für größere Portionen kann einfach eine größere Pfanne genommen werden. Dadurch kann man die ganze Familie zeitsparend satt machen.
+
+## Zutaten
+
+- 3 Eier
+- 4g Salz
+- 140g Milch
+- 130g Mehl
+- 5g Natron
```

Comment:
Leicht gehäufter Teelöffel? Ich würd's einfach in Klammern dahinter setzen ^_^;

Reply:
Laut Produktpackung entspricht es genau 1 TL